.browserslistrc

```diff
@@ -0,0 +1,13 @@
+# https://github.com/browserslist/browserslist#readme
+
+>= 1%
+last 1 major version
+not dead
+Chrome >= 45
+Firefox >= 38
+Edge >= 12
+Explorer >= 10
```

Comment:
I think there was something about IE 10 that we didn't support... but I can't recall off hand what it was...